src/Cool/StrongTypes/MathExpression.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "cereal/types/string.hpp"
+
+namespace Cool {
+
+struct MathExpression {
+    std::string expression{"x"};
+
+    friend auto operator==(const MathExpression&, const MathExpression&) -> bool = default;
+
+private:
+    // Serialization
+    friend class cereal::access;
+    template<class Archive>
+    void serialize(Archive& archive)
+    {
+        archive(cereal::make_nvp("MathExpression", expression));
+    }
+};
+
+inline auto to_string(const MathExpression& math_expression) -> std::string
+{
+    return math_expression.expression;
```

Comment:
I would add quotes around the expression to make it clearer. If you open the debug option "Commands and Registries Windows" you will see the history that uses this function to display the current and previous value of your variable 
![image](https://user-images.githubusercontent.com/45451201/214416522-6e44d50b-eeed-4fe3-8285-7fc786001ebe.png)